packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useMemo } from 'react';
+import { FlowProvider, TFlowProviderContext } from '../../../../components/FlowProvider';
+import { useModal } from '../../../../components/ModalProvider';
+import { ModalStepWrapper, WalletButton } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
+import { Onfido } from '../../screens';
+import { useAuthentication, usePOA, usePOI } from '@deriv/api';
+import { THooks } from '../../../../types';
+
+const Idv = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>IDV screen</h1>
+        </div>
+    );
+};
+
+const Manual = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>Manual screen</h1>
+        </div>
+    );
+};
+
+const Poa = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>POA screen</h1>
+        </div>
+    );
+};
+
+const PersonalDetails = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>Personal details screen</h1>
+        </div>
+    );
+};
+
+const Loading = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <Loader />
+        </div>
+    );
+};
+
+const Password = () => {
+    return <div style={{ fontSize: 60, height: 400, width: 600 }}>Password screen</div>;
+};
+
+// TODO: Replace these mock components with the screens
+const screens = {
+    idvScreen: <Idv />,
+    loadingScreen: <Loading />,
+    manualScreen: <Manual />,
+    onfidoScreen: <Onfido />,
+    passwordScreen: <Password />,
+    personalDetailsScreen: <PersonalDetails />,
+    poaScreen: <Poa />,
+};
+
+const Verification = ({ selectedJurisdiction }: { selectedJurisdiction: string }) => {
+    const { data: poiStatus, isSuccess } = usePOI();
+    const { data: poaStatus, isSuccess: isSuccessPOAStatus } = usePOA();
+    const { data: authenticationData } = useAuthentication();
+    const { hide } = useModal();
+
+    const isLoading = useMemo(() => {
+        return !isSuccess || !isSuccessPOAStatus;
+    }, [isSuccess, isSuccessPOAStatus]);
+
+    const hasAttemptedPOA = poaStatus?.has_attempted_poa || true;
+    const needPersonalDetails = true;
+
+    const initialScreenId: keyof typeof screens = useMemo(() => {
+        const service = (poiStatus?.current?.service || 'manual') as keyof THooks.POI['services'];
```

Comment:
You can use `poiStatus.previous.status` for this.
If there is, check for the `poiStatus.previous.status`and show the status screen.

Seems like im missing the status of the current service, will update the hook 🤔